File: CustomApplications/VB.Net/MarsProbe/Form2.vb
Public Class Form2

    Public Property ReportData() As String
        Get
            Return Me.richTextBox1.Text
        End Get
        Set(ByVal value As String)
            Me.richTextBox1.Text = value
        End Set
    End Property
    Public Property FormTitle() As String

        Get
            Return Me.Text
        End Get
        Set(ByVal value As String)
            Me.Text = value
        End Set
    End Property

End Class

File: CustomApplications/CSharp/HohmannTransfer/Form2.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace HohmannTransfer
{
    public partial class Form2 : Form
    {
        public Form2()
        {
            InitializeComponent();
        }

        public String ReportData
        {
            get { return this.richTextBox1.Text; }
            set { this.richTextBox1.Text = value; }
        }

        public String FormTitle
        {
            get { return this.Text; }
            set { this.Text = value; }
        }

    }
}